src/lib/DateInput.svelte

```diff
@@ -157,7 +195,13 @@
     }}
   />
   {#if visible && !disabled}
-    <div class="picker" class:visible transition:fly={{ duration: 80, easing: cubicInOut, y: -5 }}>
+    <div
+      class="picker"
+      class:visible
+      transition:fly={{ duration: 80, easing: cubicInOut, y: -5 }}
+      style={`left: ${pickerLeftPosition}px; top: ${pickerTopPosition}px;`}
```

Comment:
I'd like to avoid using inline styles if possible. Could something like this work instead?
```svelte
class:above={positioning === 'above'}
class:below={positioning === 'below'}
class:center={positioning === 'center'}
```

Reply:
I moved the variables to global css variables, and moved the styling into the css and out of the inline style. The left and right need to be set dynamically as they depend on screen size so having a static value would result in weird behavior for some screen sizes.

Reply:
It can be placed above using `bottom: 100%`, and centering.

Horizontal positioning I think you're right about, because the `input` width doesn't match `date-time-field` (which is probably due to a stupid decision by me).

The center positioning I can't seem to get working:
<img width="1156" alt="image" src="https://github.com/probablykasper/date-picker-svelte/assets/11315492/3d05146f-2c07-4bb1-a568-bf9b79da3d2c">

Reply:
I missed that `bottom: 100%` would properly place it vertically. 

I feel like its probably best to keep them the same though, instead of doing the vertical and horizontal components differently.  I think this keeps it more readable, unless you disagree?

Reply:
Using `bottom: 100%` would be simpler since there's no need to calculate any positions, so I prefer that